pkg/testutil/testutil_linux.go

```diff
@@ -33,6 +33,7 @@ var (
 	MariaDBImage                = mirrorOf("mariadb:10.5")
 	GolangImage                 = mirrorOf("golang:1.18")
 	DockerAuthImage             = mirrorOf("cesanta/docker_auth:1.7")
+	UbuntuImage                 = mirrorOf("ubuntu:20.04")
```

Comment:
ubuntu 22.04

Reply:
no 22.04 
https://github.com/orgs/stargz-containers/packages/container/ubuntu/versions :(

Reply:
Can we just use alpine?

Reply:
apart from this PR, @ktock  can we add `ubuntu 22.04` to stargz repo https://github.com/orgs/stargz-containers/packages/container/ubuntu/versions ?